alfalfa_web/server/dbops.js

```diff
@@ -93,9 +94,13 @@ function writePoint(id, siteRef, level, val, who, dur, db) {
             });
         } else {
           // In this case the point has never been written to and there is no
-          // existing write array in the db so we create a new one
+          // existing writearray in the db, so error out. A default writearray
+          // should have been created when the rec was created on the backend.
+          console.log("No writearrary found for point " + id + ". Make sure it exists in the original tag file.");
```

Comment:
`writearrays` are now to be propopulated when we parse the tags. This will allow for the frontend to only read/set values on existing records.